packages/react-core/src/components/Modal/ModalContent.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ModalBoxBody from './ModalBoxBody';
+import ModalBoxHeader from './ModalBoxHeader';
+import ModalBoxHCloseButton from './ModalBoxCloseButton';
+import ModalBox from './ModalBox';
+import ModalBoxFooter from './ModalBoxFooter';
+import Backdrop from './Backdrop';
+import Bullseye from '../../layouts/Bullseye/Bullseye';
+
+const propTypes = {
+  /** content rendered inside the Modal. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the button */
+  className: PropTypes.string,
+  /** Flag to show the modal */
+  show: PropTypes.bool,
+  /** Content of the Modal Header */
+  header: PropTypes.any,
+  /** Content of the Modal Footer */
+  footer: PropTypes.any,
+  /** A callback for when the close button is clicked */
+  onClose: PropTypes.func,
+  /** Creates a large version of the Modal */
+  isLarge: PropTypes.bool,
+  /* Aria label used for Modal Box Header */
+  label: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+  className: '',
+  show: false,
+  header: null,
+  footer: null,
+  onClose: () => undefined,
+  isLarge: false
+};
+
+const ModalContent = ({
+  children,
+  className,
+  show,
+  header,
```

Comment:
I think I would rather see this prop as `title` instead since we expect only a string for the title currently.  In the future, we can make use of header for users that want to pass in a completely custom header.

Reply:
This is related to a previous comment I made, about only defining a text string for the header. I'd like to loop @mcarrano in on this one as well. I'm making an assumption that we would only ever want text to display as the modal header, which might not be true. But even if that is true from a patternfly perspective, would we still want to enable the consumer to define a custom header?